onnx/defs/traditionalml/defs.cc

```diff
@@ -28,10 +28,10 @@ ONNX_ML_OPERATOR_SET_SCHEMA(
              "tensor(int64)",
              "tensor(int32)",
              "tensor(string)"},
-            "allowed types."));
```

Comment:
adding some description here is better.

Reply:
I agree. The one I found there was way to generic and didn't add any information. I'll go through and write something useful for each type constraint.